stackdriver/logging.ru

```diff
@@ -0,0 +1,20 @@
+require_relative "sinatra_logging"
```

Comment:
Using rackup files for Sinatra examples adds confusion.

See [Sinatra: README](http://www.sinatrarb.com/intro.html). The intro uses a simple `myapp.rb` file.

In my experience with Sinatra, I *always* start with a simple .rb file. That is idiomatic Sinatra. Frequently `app.rb` which is run simple via `ruby app.rb`

Let's only use a rackup file for the Rack examples.